### PR TITLE
[iOS] Introduce SwiftGen

### PR DESCRIPTION
**Pull request by ry-itto**
## Issue
- N/A

## Overview (Required)
- Introduce SwiftGen
  - Use images safety
  - Use color asset's color safety
  - Ready for localizing

## Links
- [SwiftGen](https://github.com/SwiftGen/SwiftGen)

### CODE REVIEW

**Comment by ry-itto** [comment]
Someone please review this PR 🙏

**Comment by roana0229** [comment]
👀

File: ios-base/Makefile [hunk 1]
```diff
@@ -11,6 +11,10 @@ xcode:
 	mint run xcodegen
 	bundle exec pod install
 
+swiftgen:
+	mkdir -p $(GIT_REPO)/ios-base/DroidKaigi\ 2020/Generated
+	mint run swiftgen swiftgen
```
Comment (roana0229) [inline ios-base/Makefile]: Could you please `SwiftGen/SwiftGen@6.1.0` add to `Mintfile` ?

Comment (ry-itto) [reply ios-base/Makefile]: Oh, sorry. I missed..
I will add it 👍

**Comment by ry-itto** [comment]
@roana0229 I fixed! 👍

**Comment by roana0229** [comment]
Thank you for quick work 👍

**Pull request merged**